app/views/managementtools/assignedUsersTable.scala.html

```diff
@@ -33,15 +34,15 @@
             <th id="@{usersTab.tabMessageKey}-case" class="bold w-0">Case</th>
         </tr>
 
-        @for((result, index) <- usersTab.searchResult.results.zipWithIndex) {
+        @for((result, index) <- usersTab.searchResult.zipWithIndex) {
 
                 <tr id="@{usersTab.tabMessageKey}-details-@index">
                     <td id="@{usersTab.tabMessageKey}-user-@index">
-                        <a href="">@result.name</a>
+                        <a href="@controllers.v2.routes.ManageUserController.displayUserDetails(result.id)">@result.name.getOrElse(result.id)</a>
```

Comment:
there is something like safeName in the Operator class that either displays the name of "PID X"